src/BottomNavigation/BottomNavigationItem.spec.js

```diff
@@ -0,0 +1,36 @@
+// @flow weak
+/* eslint-env mocha */
+
+import React from 'react';
+import { assert } from 'chai';
+import { createShallowWithContext } from 'test/utils';
+import BottomNavigationItem, { styleSheet } from './BottomNavigationItem';
+
+describe('<BottomNavigationItem>', () => {
```

Comment:
Tests are incomplete--please cover the various prop-driven className combinations